pkg/manager/manager_test.go

```diff
@@ -345,6 +354,42 @@ var _ = Describe("Manager", func() {
 			mockedNl.AssertExpectations(t)
 			mockedSr.AssertExpectations(t)
 		})
+		It("Attaching dummy link to the bridge with MTU (success)", func() {
+			origMtu := 1500
```

Comment:
This and the test above have a lot in common, can you move the common mocks and init logic to the context block ?

Reply:
I will add MTU check to the previous one.